MovieMVP/MovieMVPTests/MockRealmService.swift

```diff
@@ -0,0 +1,55 @@
+// MockRealmService.swift
+// Copyright © RoadMap. All rights reserved.
+
+@testable import MovieMVP
+import RealmSwift
+
+/// Моковый Реалм сервис
+final class MockRealmService: RealmServicable {
```

Comment:
```suggestion
final class MockRealmService: RealmServicable {
```
а в чем тут смысл Мока если запрос в реальный реалм откправляется

Reply:
тогда уберу его, я его ночью делал, не понял до конца как с ним разобраться